libnmstate/ifaces/ifaces.py

```diff
@@ -165,6 +166,43 @@ def _validate_ovs_patch_peers(self):
                             " patch port"
                         )
 
+    def _validate_vlan_mtu_lte_base_mtu(self):
+        """
+            Validate that mtu of vlan or vxlan is less than
+            or equal to it's base interface's MTU
+
+            If base MTU is not present, set same as vlan MTU
+        """
+        for iface in self._ifaces.values():
+            if (
+                iface.iface_type == InterfaceType.VLAN
+                or iface.iface_type == InterfaceType.VXLAN
+            ):
+
+                if iface.state == InterfaceState.UP:
+
+                    if not iface.mtu:
+                        raise NmstateValueError(
+                            f"paramater mtu missing for vlan {iface.name}"
+                        )
+
+                    base_iface = self._ifaces.get(iface.parent)
+                    if not base_iface:
+                        raise NmstateValueError(
+                            f"base interface {iface.parent} for vlan \
+                                            {iface.name} does not exist"
+                        )
+
+                    if not base_iface.mtu:
+                        base_iface.mtu = iface.mtu
+
+                    if iface.mtu > base_iface.mtu:
+                        raise NmstateValueError(
+                            f"{iface.name} mtu {iface.mtu} cannot be \
```

Comment:
The use of `\` is incorrect: by using it, you add a lot spaces.

You should use:

```python
raise NmstateValueError(
    f"Interface {iface.name} has bigger "
    f"MTU({iface.mtu}) "
    f"than its base interface: {iface.parent} "
    f"MTU({parent_iface.mtu})"
)
```